lib/ansiblelint/__init__.py

```diff
@@ -22,7 +22,6 @@
 from collections import defaultdict
 import logging
 
-from ansiblelint.rules import AnsibleLintRule  # noqa F401: exposing public API
```

Comment:
Let's avoid relocating this to the bottom.

Reply:
That is by design because otherwise you will get circular imports failure:
```
ImportError while loading conftest '/Users/ssbarnea/c/os/ansible-lint/test/conftest.py'.
test/__init__.py:7: in <module>
    from ansiblelint.runner import Runner
lib/ansiblelint/__init__.py:28: in <module>
    from ansiblelint.rules import AnsibleLintRule  # noqa F401: exposing public API
lib/ansiblelint/rules/__init__.py:124: in <module>
    from ansiblelint import RulesCollection  # noqa F401: new class location
E   ImportError: cannot import name 'RulesCollection' from partially initialized module 'ansiblelint' (most likely due to a circular import) (/Users/ssbarnea/c/os/ansible-lint/lib/ansiblelint/__init__.py)
FAIL: 4
```